src/snapshots/bound_type_var_no_annotation.md

```diff
@@ -35,9 +35,9 @@ main! = |_| {
 UNUSED VARIABLE - bound_type_var_no_annotation.md:19:5:19:9
 # PROBLEMS
 **UNUSED VARIABLE**
-Variable ``pair`` is not used anywhere in your code.
+Variable `pair` is not used anywhere in your code.
 
-If you don't need this variable, prefix it with an underscore like `_pair` to suppress this warning.
+If you don't need this variable, prefix it with an underscore like `_pair to suppress this warning.
```

Comment:
missed backtick here an in other snaphsots

Reply:
Ok, I think I got them all now 😅